src/app/helptext/jails/jails-add.ts

```diff
@@ -356,6 +362,9 @@ target="_blank">ZFS(8)</a> man page has information \
 on how to configure \
 the ZFS filesystem to operate from within a jail.'),
 
+allow_vmm_placeholder: T('allow_vmm'),
+allow_vmm_tooltip: T(''),
```

Comment:
Allow the jail to access the bhyve virtual machine monitor (VMM). The jail must have FreeBSD 12.0 or newer installed with the <a href="https://www.freebsd.org/cgi/man.cgi?query=vmm" target="_blank">vmm(4)</a> kernel module loaded.